docs/conceptual/More-about-how-ROCm-uses-PCIe-Atomics.rst

```diff
@@ -1,156 +1,63 @@
-bridges need to have atomic operations routing enabled or the atomic operations will fail even though
-PCIe endpoint and PCIe I/O devices has the capability to atomic operations.
+To learn more about the industry standards and specifications of PCIe, see `PCI-SIG Specification <https://pcisig.com/specifications>`_.
 
-To do atomic operations routing capability between two or more Root Ports, each associated Root Port
-must indicate that capability via the atomic operations routing supported bit in the DevCap2 register.
+To learn more about PCIe and its capabilities, consult the following white papers:
 
-If your system has a PCIe Express Switch it needs to support atomic operations routing. Atomic
-operations requests are permitted only if a component's ``DEVCTL2.ATOMICOP_REQUESTER_ENABLE``
-field is set. These requests can only be serviced if the upstream components support atomic operation
-completion and/or routing to a component which does. Atomic operations routing support=1, routing
-is supported; atomic operations routing support=0, routing is not supported.
+* `Atomic Read Modify Write Primitives by Intel <https://www.intel.es/content/dam/doc/white-paper/atomic-read-modify-write-primitives-i-o-devices-paper.pdf>`_
+* `PCI Express 3 Accelerator White paper by Intel <https://www.intel.sg/content/dam/doc/white-paper/pci-express3-accelerator-white-paper.pdf>`_
+* `PCIe Generation 4 Base Specification includes atomic operations <https://astralvx.com/storage/2020/11/PCI_Express_Base_4.0_Rev0.3_February19-2014.pdf>`_
+* `Xilinx PCIe Ultrascale White paper <https://docs.xilinx.com/v/u/8OZSA2V1b1LLU2rRCDVGQw>`_
 
-An atomic operation is a non-posted transaction supporting 32-bit and 64-bit address formats, there
-must be a response for Completion containing the result of the operation. Errors associated with the
-operation (uncorrectable error accessing the target location or carrying out the atomic operation) are
-signaled to the requester by setting the Completion Status field in the completion descriptor, they are
-set to to Completer Abort (CA) or Unsupported Request (UR).
+Working with PCIe 3.0 in ROCm
+-------------------------------
+Starting with PCIe 3.0, atomic operations can be requested, routed through, and completed by PCIe components. Routing and completion do not require software support. Component support for each can be identified by the Device Capabilities 2 (DevCap2) register. Upstream
+bridges need to have atomic operations routing enabled. If not enabled, the atomic operations will fail even if the 
+PCIe endpoint and PCIe I/O devices can perform atomic operations. 
 
-To understand more about how PCIe atomic operations work, see
-`PCIe atomics <https://pcisig.com/specifications/pciexpress/specifications/ECN_Atomic_Ops_080417.pdf>`_
+If your system uses PCIe Express Switch to connect and enable communication between multiple PCIe components, the switch must also support atomic operations routing.
```

Comment:
"PCIe Express" is redundant since the "e" stands for "express". And there is a missing word here. This should either be "If your system uses PCIe switches ... the switches must also ..." or "If your system uses a PCIe switch ..."